changelog.txt

```diff
@@ -21,7 +21,8 @@
 * Instant Search: improve accessibility of the Search modal.
 * Likes: remove jQuery dependency where possible.
 * Pay with PayPal Block: improve the styling of the buttons in subscription emails.
-* Pay with Paypal Block: improve the display of currencies.
+* Pay with PayPal Block: improve the display of currencies.
+

```

Comment:
Oops, we added an extra linebreak here.

I think Github's "insert a suggestion" UI added it and I forgot to delete it in this case

```suggestion
```